### PR TITLE
cli enhancement

### PR DESCRIPTION
**Pull request by santoshkumarradha**
Adding basic formating, 

- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by santoshkumarradha** [comment]
@wjcunningham7 amazing !

one small error when checking this out
`covalent status` the first time without anything running after installing covalent gives me this 

```
@santoshkumarradha ➜ /workspaces/covalent (santosh/cli-enhancement) $ covalent status
╭────────────────────────────────────────────────╮
│                                                │
│                    Covalent                    │
│                                                │
╰────────────────────────────────────────────────╯
GitHub: https://github.com/AgnostiqHQ/covalent
Docs:   https://docs.covalent.xyz

╭────────────────╮
│ Service Status │
╰────────────────╯

Traceback (most recent call last):
  File "/home/codespace/.python/current/bin/covalent", line 8, in <module>
    sys.exit(cli())
  File "/home/codespace/.local/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/home/codespace/.local/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/home/codespace/.local/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/codespace/.local/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/codespace/.local/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/workspaces/covalent/covalent_dispatcher/_cli/service.py", line 634, in status
    admin_address = _get_cluster_admin_address()
  File "/workspaces/covalent/covalent_dispatcher/_cli/service.py", line 849, in _get_cluster_admin_address
    admin_host = get_config("dask.admin_host")
  File "/workspaces/covalent/covalent/_shared_files/config.py", line 253, in get_config
    return cm.get(entries)
  File "/workspaces/covalent/covalent/_shared_files/config.py", line 176, in get
    return reduce(getitem, key.split("."), self.config_data)
KeyError: 'admin_host'
```

**Comment by santoshkumarradha** [comment]
GTG sir !

**Comment by santoshkumarradha** [comment]
@wjcunningham7 this is gtg but since i created the PR, i am unable to approve lol.

File: requirements.txt [hunk 1]
```diff
@@ -6,11 +6,13 @@ cloudpickle>=2.0.0
 dask[distributed]>=2022.6.0
 fastapi>=0.93.0
 furl>=2.1.3
+natsort>=8.4.0
```
Comment (santoshkumarradha) [inline requirements.txt:9]: @wjcunningham7 is this being used anywhere ?

Comment (wjcunningham7) [reply requirements.txt:9]: It's used to sort the names of the dask workers when running `covalent cluster --status`

**Review by FyzHsn — DISMISSED** [review]

**Comment by santoshkumarradha** [comment]
@FyzHsn removing rich from reqs seems to make the rtd work.

**Comment by santoshkumarradha** [comment]
OKay having `~=` for rich version seems to work, might need to find the correct `>=` for that by trial and error

**Comment by FyzHsn** [comment]
> OKay having `~=` for rich version seems to work, might need to find the correct `>=` for that by trial and error

Thank you for the excellent lead sir! I'll play with this.

**Review by FyzHsn — APPROVED** [review]

**Pull request merged**